test/parallel/test-child-process-fork-stdio.js

```diff
@@ -50,5 +50,6 @@ if (process.argv[2] === 'child') {
     assert.strictEqual(stderr, 'this should not be ignored');
   }));
 
+  // TODO: stdio[4] is not writable?
   child.stdio[4].write(buf);
```

Comment:
@cjihrig @addaleax I'm unsure how this is supposed to work? `stdio[4]` is set as `!writable`? I think there is a bug here which this PR hits.